### DISCUSSION TITLE
安装最新版之后专业版变成免费版了

### DISCUSSION THREAD

**Discussion by WaldenLi** [getlantern/lantern-forum-cn#447] [General]
window10系统，邮箱591117678@qq.com , 之前一直闪退，今天安装最新版之后可以打开了，但是变成免费版了 。
有试过卸载，删除文件夹重装，还是一样
![image](https://github.com/getlantern/lantern-forum-cn/assets/12380519/059d4444-a373-448d-a6aa-63060d9f407b)
![image](https://github.com/getlantern/lantern-forum-cn/assets/12380519/cb3d38fc-a8cb-41aa-a0b2-4c7d2e1bae9f)

**Comment by MaxMKMax** [comment]
@WaldenLi 
您好. 我已经帮您提交问题上去了. 您也可以试看重新登录.
